package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "open-sesame",
+  "version": "1.0.0",
+  "description": "Open Sesame is a platform intended to make contributing to open source less intimidating by matching first-time contributors to experienced mentors.",
+  "main": "index.js",
+  "scripts": {
+    "dev-server": "npm run jsx-preprocessor && mvn package appengine:run || true",
+    "jsx-preprocessor": "npm run jsx-projects && npm run jsx-common",
+    "jsx-projects": "npx babel src/main/webapp/js/projects/react/ --minified --ignore jsx-processed/ --out-dir src/main/webapp/js/projects/react/jsx-processed --presets react-app/prod",
+    "jsx-projects-live": "npx babel --watch src/main/webapp/js/projects/react/ --minified --ignore jsx-processed/ --out-dir src/main/webapp/js/projects/react/jsx-processed --presets react-app/prod",
+    "jsx-common": "npx babel src/main/webapp/js/common/react/ --minified --ignore jsx-processed/ --out-dir src/main/webapp/js/common/react/jsx-processed --presets react-app/prod"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/clayferg/clayferg-projects.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/clayferg/clayferg-projects/issues"
+  },
+  "homepage": "https://github.com/clayferg/clayferg-projects#readme",
+  "dependencies": {
```

Comment:
Add "eslint": "^6.8.0" to dependencies

https://medium.com/@stalonadsl948/dependencies-vs-devdependencies-926e096a3dee

Reply:
Cool, added in the commit below.